sympy/matrices/matrices.py

```diff
@@ -3452,6 +3452,14 @@ def norm(self, ord=None):
                 # Minimum singular value
                 return Min(*self.singular_values())
 
+            elif ord == S.Infinity:   # Infinity Norm - Maximum row sum
+                m = self.applyfunc(abs)
+                return Max(*[sum(m.row(i)) for i in range(m.rows)])
+
+            elif ord == S.NegativeInfinity: # Negative Infinity Norm - Maximum column sum
```

Comment:
The "negative infinity norm" or matrices would not be maximum column sum. The maximum column sum is 1-norm, already implemented above.

If we want to introduce -oo norm for matrices, that merits a separate discussion. For example, [Matlab docs](https://www.mathworks.com/help/symbolic/norm.html?s_tid=gn_loc_drop) do not define it. 

Generally, one may think  that the order -p norm for matrices should be min||Ax||_p over ||x||_p = 1 (this is how it works for p=2 currently) but this is just one interpretation.